src/unix/core.c

```diff
@@ -1226,3 +1227,26 @@ void uv_os_free_passwd(uv_passwd_t* pwd) {
 int uv_os_get_passwd(uv_passwd_t* pwd) {
   return uv__getpwuid_r(pwd);
 }
+
+/* Leverage RFC1323: TCP extensions for high performance
```

Comment:
I think the info in this comment would be more useful in the doc for the API than in the source.